pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -645,6 +647,11 @@ def _get_file(remote_file, data_path, fname, temp_path=None, zip_method=None):
     if zip_method == 'zip':
         with zipfile.ZipFile(dl_fname, 'r') as open_zip:
             open_zip.extractall(data_path)
+    elif zip_method == 'gz':
+        dest = os.path.join(data_path, fname.replace('.gz',''))
```

Comment:
What is the functionality or issue that gzip is handling that zipfile package does not?

Reply:
Yes, I tried using the zip package to unzip the gzip files but I wasn't able to since these are for other compression library, but if you could take a look would be best.